<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewArtworkWizard</class>
 <widget class="QWizard" name="NewArtworkWizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>686</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Artwork</string>
  </property>
  <property name="options">
   <set>QWizard::HaveFinishButtonOnEarlyPages</set>
  </property>
  <widget class="QWizardPage" name="wizardPage">
   <property name="title">
    <string>Select the Customer / Product</string>
   </property>
   <property name="subTitle">
    <string>Step 1:</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="CustomQueryView" name="lstCustomers"/>
     </widget>
    </item>
    <item>
     <widget class="CustomQueryView" name="lstProducts"/>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="wizardPage_2">
   <property name="title">
    <string>Artwork Details</string>
   </property>
   <property name="subTitle">
    <string>Enter the artwork dimensions and other details:</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="4" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>114</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Film Dimensions (Calculated)</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="2">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Total Sq. Inches:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QLineEdit" name="txtFilmTotalSqInch">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="txtFilmLength">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Length:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="txtFilmSingleSqInch">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Sq. Inches Per Film:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Width:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLineEdit" name="txtFilmWidth">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Film Dimensions (Entered)</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Length:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Width:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Sq. Inches Per Film:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="txtFilmSingleSqInchEntered">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QLineEdit" name="txtFilmTotalSqInchEntered">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Total Sq. Inches:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="txtFilmLengthEntered">
         <property name="enabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLineEdit" name="txtFilmWidthEntered">
         <property name="enabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Size (mm):</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="txtArtworkSize"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Number Of Colors:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="spinNoOfColors"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Detected Paper Size:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="txtDetectedPaperSize">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>User Paper Size:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="cboUserPaperSize">
        <item>
         <property name="text">
          <string>A4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>A4 Extra</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>A3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>A3 Extra</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>A2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>A3 Extra</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>A1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>A1 Extra</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>A0</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QCheckBox" name="chklChargeable">
      <property name="text">
       <string>Chargeable</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="wizardPage_3">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>142</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCommandLinkButton" name="cmdSave">
        <property name="text">
         <string>Save</string>
        </property>
        <property name="description">
         <string>Click here to save the artwork. Or click on Finish</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>142</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomQueryView</class>
   <extends>QTreeView</extends>
   <header>customqueryview.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>txtArtworkSize</tabstop>
  <tabstop>spinNoOfColors</tabstop>
  <tabstop>txtDetectedPaperSize</tabstop>
  <tabstop>cboUserPaperSize</tabstop>
  <tabstop>txtFilmLength</tabstop>
  <tabstop>txtFilmWidth</tabstop>
  <tabstop>txtFilmSingleSqInch</tabstop>
  <tabstop>txtFilmTotalSqInch</tabstop>
  <tabstop>txtFilmLengthEntered</tabstop>
  <tabstop>txtFilmWidthEntered</tabstop>
  <tabstop>txtFilmSingleSqInchEntered</tabstop>
  <tabstop>txtFilmTotalSqInchEntered</tabstop>
  <tabstop>chklChargeable</tabstop>
  <tabstop>cmdSave</tabstop>
  <tabstop>lstProducts</tabstop>
  <tabstop>lstCustomers</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
